docs/releases/ez_platform_v1.13.0_lts.md

```diff
@@ -0,0 +1,65 @@
+# eZ Platform v1.13.0
+
+**The Long Term Support v1.13.0 release of eZ Platform and eZ Platform Enterprise Edition is available as of December 22, 2017.**
+
+## Notable changes since v1.12.0
+
+### Link manager
+
+The new Link manager enables you to manage all links to external websites that are embedded in the whole site,
```

Comment:
Is "Link Manager" a feature that we can spell with first Uppercased letters?

Reply:
It's lowercase because that is the spelling that is used in the interface.